tasks/section5.yml

```diff
@@ -351,15 +489,19 @@
       - rule_5.2.10
 
 - name: "SCORED | 5.2.10 | PATCH | Ensure SSH PermitUserEnvironment is disabled"
-  command: /bin/true
+  lineinfile:
+      state: present
+      dest: /etc/ssh/sshd_config
+      regexp: '^PermitUserEnvironment'
+      line: 'PermitUserEnvironment no'
   tags:
       - level1
       - level2
       - patch
       - rule_5.2.10
 
 - name: "SCORED | 5.2.11 | AUDIT | Ensure only approved ciphers are used"
-  command: /bin/true
+  command: grep "Ciphers aes128-ctr,aes192-ctr,aes256-ctr" /etc/ssh/sshd_config
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*